include/RAJA/policy/cuda/atomic.hpp

```diff
@@ -47,12 +47,13 @@ namespace atomic
 namespace detail
 {
 
-#if (__GNUC__ || __clang__ || __xlC__) && !__CUDA_ARCH__  // begin host compiler check
-// Silencing extraneous warnings when host compiler is activated (without device compiler) within nvcc.
-#else  // using nvcc compiler
+#if __CUDA_ARCH__ && __CUDA_ARCH__ < 350  // baseline CUDA_ARCH sm_35 check
```

Comment:
should this be # if defined(__CUDA_ARCH__)

Reply:
Done, thanks.